UT4MasterServer.Web/src/pages/Profile/PlayerCard.vue

```diff
@@ -66,21 +66,5 @@ img.avatar {
 </style>
 
 <script lang="ts" setup>
-import { SessionStore } from '../../stores/session-store';
-import { onMounted, shallowRef } from 'vue';
-import CustomService from '../../services/custom.service';
-import { IPlayerCard } from '../../types/player-card';
-
-const customService = new CustomService();
-
-const playerCard = shallowRef<IPlayerCard | undefined>(undefined);
-
-async function fetchProfile() {
-  if (SessionStore.session) {
-    playerCard.value = await customService.getPlayerCard(SessionStore.session.account_id);
-  }
-}
-
-onMounted(fetchProfile);
-
+import { AccountStore } from '@/stores/account-store';
 </script>
```

Comment:
We will use `playerCard` to also show other players cards in game.
IMO we should have deleted `playerCard` constant here, to be able to modify it when we get player ID from GET (original Epic URL: https://www.epicgames.com/unrealtournament/en-US/playerCard?playerId=63e4964c6cc1441089a726c00fcb2926)